src/frontend/screens/Settings/components/AutoCloseLauncher.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { ToggleSwitch } from 'frontend/components/UI'
+import useSetting from 'frontend/hooks/useSetting'
+
+const AutoCloseLauncher = () => {
+  const { t } = useTranslation()
+  const [autoCloseLauncher, setAutoCloseLauncher] = useSetting(
+    'autoCloseLauncher',
+    false
+  )
+
+  return (
+    <ToggleSwitch
+      htmlId="autoCloseLauncher"
+      value={autoCloseLauncher}
+      handleChange={() => setAutoCloseLauncher(!autoCloseLauncher)}
+      title={t(
+        'setting.auto-close-launcher',
+        'Close Heroic After Game Launch ⚠️ Disables Cloud Async'
```

Comment:
Maybe that should be Disables `Cloud Saves` or `Cloud Sync`

Reply:
yup